data/json/overmap_terrain.json

```diff
@@ -6887,5 +6887,197 @@
+    "mondensity": 2,
+    "extras": "build",
+    "flags": [ "SIDEWALK" ]
+  },
+  {
+    "id": "small_wooded_trail",
+    "type": "overmap_terrain",
+    "name": "small wooded trail",
+    "sym": 83,
+    "color": "i_green",
+    "see_cost": 5,
+    "mondensity": 2,
+    "extras": "build",
+    "flags": [ "SIDEWALK" ]
+  },
+  {
+    "id": "art_gallery",
+    "type": "overmap_terrain",
+    "name": "art gallery",
+    "sym": 97,
+    "color": "i_yellow",
+    "see_cost": 5,
+    "mondensity": 2,
+    "extras": "build",
+    "flags": [ "SIDEWALK" ]
+  },
+  {
+    "id": "derelict_property",
+    "type": "overmap_terrain",
+    "name": "derelict property",
+    "sym": 88,
+    "color": "i_brown",
+    "see_cost": 5,
+    "mondensity": 2,
+    "extras": "field"
+  },
+  {
+    "id": "state_park_0_0",
+    "type": "overmap_terrain",
+    "name": "state park",
+    "sym": 83,
+    "color": "i_green",
+    "see_cost": 5,
+    "mondensity": 2
+  },
+  {
+    "id": "state_park_0_1",
+    "type": "overmap_terrain",
+    "name": "state park",
+    "sym": 83,
+    "color": "i_green",
+    "see_cost": 5,
+    "mondensity": 2
+  },
+  {
+    "id": "state_park_1_0",
+    "type": "overmap_terrain",
+    "name": "state park parking",
+    "sym": 36,
+    "color": "i_green",
+    "see_cost": 5,
+    "mondensity": 2,
+    "flags": [ "SIDEWALK" ]
+  },
+  {
+    "id": "state_park_1_1",
+    "type": "overmap_terrain",
+    "name": "state park",
+    "sym": 83,
+    "color": "i_green",
+    "see_cost": 5,
+    "mondensity": 2
+  },
+  {
+    "id": "pavilion",
+    "type": "overmap_terrain",
+    "name": "pavilion",
+    "sym": 65,
+    "color": "i_green",
+    "see_cost": 5,
+    "mondensity": 2,
+    "flags": [ "SIDEWALK" ]
+  },
+  {
+    "id": "fishing_pond_0_0",
+    "type": "overmap_terrain",
+    "name": "fishing pond",
+    "sym": 83,
+    "color": "i_blue",
+    "see_cost": 5,
+    "mondensity": 2
+  },
+  {
+    "id": "fishing_pond_0_1",
+    "type": "overmap_terrain",
+    "name": "fishing pond",
+    "sym": 83,
+    "color": "i_blue",
+    "see_cost": 5,
+    "mondensity": 2
+  },
+  {
+    "id": "fishing_pond_1_0",
+    "type": "overmap_terrain",
+    "name": "fishing pond",
+    "sym": 83,
+    "color": "i_blue",
+    "see_cost": 5,
+    "mondensity": 2,
+    "flags": [ "SIDEWALK" ]
+  },
+  {
+    "id": "fishing_pond_1_1",
+    "type": "overmap_terrain",
+    "name": "fishing pond",
+    "sym": 83,
+    "color": "i_blue",
+    "see_cost": 5,
+    "mondensity": 2
+  },
+  {
+    "id": "hunting_blind",
+    "type": "overmap_terrain",
+    "name": "hunting blind",
+    "sym": 72,
+    "color": "i_green",
+    "see_cost": 5,
+    "mondensity": 2,
+    "extras": "field"
+  },
+  {
+    "id": "small_storage_units",
+    "type": "overmap_terrain",
+    "name": "small storage units",
+    "sym": 35,
+    "color": "i_yellow",
+    "see_cost": 5,
+    "mondensity": 2,
+    "extras": "build",
+    "flags": [ "SIDEWALK" ]
+  },
+  {
+    "id": "cemetary",
```

Comment:
Those are too similar to the big cemetery id. Should probably be re-id'd to "cemetery_small" or something.